### PR TITLE
Add mvaQualBDT weights

### PR DESCRIPTION
**Pull request by pmeridian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# RecoMTD-TimingIDTools
```
Comment (slava77) [inline README.md:1]: please do not remove the README and instead please add some text to describe the files currently kept in this repository

Comment (pmeridian) [reply README.md:1]: Done

**Comment by smuzaffar** [comment]
+externals

**Comment by silviodonato** [comment]
+1

**Comment by silviodonato** [comment]
I'm not sure the relevant L2s have received the notification
upgrade:  @kpedro88
reconstruction:  @perrotta @slava77

**Comment by kpedro88** [comment]
+upgrade

**Comment by mrodozov** [comment]
+externals

**Comment by mrodozov** [comment]
ah it's been approved already never mind

**Comment by slava77** [comment]
+1

this is ready to go based on the tests with the cmssw PR

**Pull request merged**